datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -0,0 +1,143 @@
+  value:
+    type: string
+    example: service
+- name: metrics
+  required: true
+  description: |
+    List of metrics to be fetched from the prometheus endpoint, if there's a
+    value it'll be renamed. This list should contain at least one metric.
+  value:
+    type: array
+    example:
+      - processor:cpu
+      - memory:mem
+      - io
+    items:
+      type: string # TODO: metrics can be a string or a key:valueß
+- name: prometheus_metrics_prefix
+  description: <PREFIX> for exposed Prometheus metrics.
+  value:
+    type: string
+    example: <PREFIX>_
+    default: null
+- name: health_service_check
+  description: |
+    Send a service check reporting about the health of the prometheus endpoint.
+    It's named <NAMESPACE>.prometheus.health
+  value:
+    type: boolean
+    example: true
+- name: label_to_hostname
+  description: Override the hostname with the value of one label.
+  value:
+    type: string
+    example: <LABEL>
+- name: label_joins
+  description: |
+    The label join allows to target a metric and retrieve it's label via a 1:1 mapping.
+  value:
+    example:
+      target_metric:
+        label_to_match: <MATCHED_LABEL>
+        labels_to_get:
+          - <EXTRA_LABEL_1>
+          - <EXTRA_LABEL_2>
+    type: object
+    properties:
+    - name: target_metric
+      type: object
+      properties:
+      - name: label_to_match
+        type: string
+      - name: labels_to_get
+        type: array
+        items:
+          type: string
+- name: labels_mapper
+  description: |
+    The label mapper allows you to rename labels.
+    Format is <LABEL_TO_RENAME>: <NEW_LABEL_NAME>
+  value:
+    example:
+      flavor: origin
+    type: object
+- name: type_overrides
+  description: |
+    Type override allows you to override a type in the prometheus payload
+    or type an untyped metrics (they're ignored by default).
+    Supported <METRIC_TYPE> are `gauge`, `counter`, `histogram`, `summary`
+  value:
+    example:
+      <METRIC_NAME>: <METRIC_TYPE>
+    type: object
+- name: send_histograms_buckets
+  description: Set send_histograms_buckets to true to send the histograms bucket.
+  value:
+    example: true
+    type: boolean
+- name: send_distribution_buckets
+  description: |
+    Set `send_distribution_buckets` to `true` to send histograms as Datadog distribution metrics.
+    
+    Learn more about distribution metrics: https://docs.datadoghq.com/developers/metrics/distributions/
+  value:
+    example: false
+    type: boolean
+- name: send_monotonic_counter
+  description: Set send_monotonic_counter to true to send counters as monotonic counter.
+  value:
+    example: true
+    type: boolean
+- name: send_distribution_counts_as_monotonic
+  description: |
+    Set send_distribution_counts_as_monotonic to true to send histograms & summary
+    counters as monotonic counters (instead of gauges).
+  value:
+    example: true
+    default: false
```

Comment:
Hmm?

Reply:
https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example#L79-L84

The original example also had this, should I change it to the same as default?